static/app/components/charts/utils.tsx

```diff
@@ -171,7 +171,8 @@ const seriesAPILadder = new GranularityLadder([
   [SIXTY_DAYS, '1d'],
   [THIRTY_DAYS, '4h'],
   [SIX_HOURS, '1h'],
-  [0, '5m'],
+  [ONE_HOUR, '5m'],
```

Comment:
how did we come up with this definition? I am just trying to understand :)

Reply:
I tried `[ONE_HOUR, '1m']` and the data looks good too. @gggritso do you know why the settings are set this way?

Reply:
Not sure if I understood the question, but I noticed a weird property of the stats_v2 endpoint:

If I use an interval of `1h` on [this query](https://sentry.io/api/0/organizations/sentry/stats_v2/?category=attachment&end=2024-11-03T14%3A00%3A34Z&field=sum%28quantity%29&groupBy=outcome&groupBy=reason&interval=1m&start=2024-11-03T13%3A35%3A38Z&utc=true), I get a result, but if I set it to anything lower, [I get empty `groups`](https://sentry.io/api/0/organizations/sentry/stats_v2/?category=attachment&end=2024-11-03T14%3A00%3A34Z&field=sum%28quantity%29&groupBy=outcome&groupBy=reason&interval=1m&start=2024-11-03T13%3A35%3A38Z&utc=true).

Then if I replace the `start` and `end` with a `statsPeriod`, I [suddenly get data again](https://sentry.io/api/0/organizations/sentry/stats_v2/?category=attachment&field=sum%28quantity%29&groupBy=outcome&groupBy=reason&interval=1m&utc=true&statsPeriod=1h).

So I don't understand under what conditions the API actually supports sub-hour granularity. Maybe it's a bug.

Reply:
@priscilawebdev nope, no idea why they're set this way! I converted the code without changing the behaviour in https://github.com/getsentry/sentry/pull/60374/. If the API supports different granularities, feel free to change them!

Sounds like maybe the API has some weird limitations?